Evergreen/Apps/Get-Fusion360.ps1

```diff
@@ -0,0 +1,37 @@
+Function Get-Fusion360 {
```

Comment:
Rename function to `Get-AutodeskFusion360` and file to `Get-AutodeskFusion360.ps1`